cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -160,15 +156,15 @@
                     <a class="m-social-media_icon"
                        href="{{ link.share_url if link.name != 'email' else link.share_url | trim }}"
                        {{ 'target="_blank" rel="noopener noreferrer"' | safe if link.name != 'email' else '' }}
-                       aria-label="Share on {{ link.name }}">
+                       aria-label="{{ _("Share on") }} {{ link.name }}">
```

Comment:
If the quoting in the translated text doesn't matter I'd make it single quotes so it isn't double quotes inside double quotes.